src/class-elements/private-method-multiple-instance-identity-same.case

```diff
@@ -0,0 +1,27 @@
+// Copyright (C) 2018 Jaideep Bhoosreddy (Bloomberg LP). All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+desc: Multiple class instances of PrivateName Private method have same function identity 
+info: |
+  Static Semantics: PrivateStringValue
+    PrivateName::
+      #IdentifierName
+  Return the String value consisting of the sequence of code units corresponding to PrivateName
+
+template: productions
```

Comment:
Please use the "default" templates, as "productions" will render tests that will contain a bunch of unnecessary and irrelevant code.